packages/compass-indexes/src/components/search-indexes-modals/base-search-index-modal.tsx

```diff
@@ -141,6 +163,17 @@ export const BaseSearchIndexModal: React.FunctionComponent<
     onSubmit(indexName, indexDefinitionDoc);
   }, [onSubmit, parsingError, indexName, indexDefinition]);
 
+  const onChangeTemplate = useCallback(
+    (template: SearchTemplate) => {
+      rafraf(() => {
+        editorRef.current?.focus();
+        editorRef.current?.applySnippet(template.snippet);
+        editorRef.current?.prettify();
```

Comment:
Why snippets are not formatted already in the constants? Running prettify can mess up with apply selection range focus and there is generally no guarantee that snippet placeholders are valid JS, so this might just not do anything

Reply:
This was actually is a scenario that Basit and I were testing. We were checking if prettify would help at this point, but you already answered, so I'm removing it.